package.json

```diff
@@ -37,6 +37,7 @@
     "webpack-dev-server": "^4.11.1",
     "webpack-merge": "5"
   },
+  "packageManager": "yarn@1.22.19",
```

Comment:
Jelenleg kétszer van bent az aktuális `yarn` verzió a `package.json` -ban. Ez annyira nem elegáns, pl egy Dokku deploy (valószínűleg egy sima Heroku deploy is) warning-ot dob rá.

- A "packageManager" kulcsot mozgassuk az "engines" alatti sorba közvetlenül.
- Töröljük ki az "engines"/"yarn" kulcsot.